tensorboard/main.py

```diff
@@ -51,12 +51,6 @@ def run_main():
   program.setup_environment()
 
   if getattr(tf, '__version__', 'stub') == 'stub':
-    # Unless the user has explicitly requested running without TensorFlow by setting the
-    # TENSORBOARD_NO_TF environment variable, we check for TensorFlow here so that if it's
-    # missing we generate a clear and immediate error rather than partial functionality.
-    # TODO(#2027): Remove environment check once we have placeholder UI
-    if os.getenv('TENSORBOARD_NO_TF') is None:
```

Comment:
If this removal is included in this PR, then we can’t merge this PR
until #2032 is merged. If you don’t want to block on that PR, then it
looks to me like you should be able to add `TENSORBOARD_NO_TF=1` to the
Travis CI script and/or Pip package smoke test as appropriate. Or, feel
free to keep this as is if you don’t mind waiting for #2032.

Reply:
For now I'll keep it, but we can remove it if this is ready to land before the UI one. Thanks!